src/components/dataContainer.js

```diff
@@ -15,9 +15,21 @@
           authProfile,
           redirectWithoutResult,
           showError,
+          currentRecord,
```

Comment:
How can I test the `currentRecord` option? Do you have a steps to reproduce? :)

Reply:
Set the new global interaction that we have added. It looks like this:

You have a DataTable and a DataContainer. **BOTH** have should have their model option configured.

Next set up this interaction

[DataTable] OnRowClick
[InteractionType] Global
[Interaction] SetCurrentRecord
[Source] DataTable.currentOption
[Target] DataContainer.currentOption

press `Play` now you should see that the DataContainer's currentRecord is changing to match the row that was clicked!

Reply:
Ah cool!